trade-finance-manager-ui/server/controllers/utilisation-reports/unlink-payment-fees/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Response } from 'express';
+import { CustomExpressRequest } from '../../../types/custom-express-request';
+import { getEditPaymentsCheckboxIdsFromObjectKeys, getFeeRecordIdsFromEditPaymentsCheckboxIds } from '../../../helpers/edit-payments-table-checkbox-id-helper';
+import { UnlinkPaymentFeesFormRequestBody } from '../../../helpers/unlink-payment-fees-helper';
+
+export type UnlinkPaymentRequest = CustomExpressRequest<{
+  reqBody: UnlinkPaymentFeesFormRequestBody;
+}>;
+
+export const postUnlinkPaymentFees = (req: UnlinkPaymentRequest, res: Response) => {
```

Comment:
The term 'UnlinkPaymentFees' is a little unclear, it would be clearer and match the terminology used on the page closer to be named 'RemoveFeesFromPayment'. Leaving the comment in just the one place but if you could update this everywhere it is used please. (e.g. the above type, and the type of the body of the above type should also change)

Reply:
Yep I agree, replaced all unlink references (including file paths) in 804824c5ea40a4ca9c021c4dff0c9b7cc0097810